maintenance/disposeOutdatedEntities.php

```diff
@@ -106,6 +112,11 @@ public function execute() {
 		$outdatedDisposer->setMessageReporter( $this->messageReporter );
 		$outdatedDisposer->run();
 
+		if ( $this->hasOption( 'with-maintenance-log' ) ) {
+			$maintenanceLogger = $maintenanceFactory->newMaintenanceLogger( 'DisposeOutdatedEntitiesLogger' );
+			$maintenanceLogger->log( $maintenanceHelper->transformRuntimeValuesForOutput() );
```

Comment:
`transformRuntimeValuesForOutput` no longer exists which got replaced by something like:

```
			$runtimeValues = $maintenanceHelper->getRuntimeValues();

			$log = [
				'Memory used' => $runtimeValues['memory-used'],
				'Time used' => $runtimeValues['humanreadable-time']
			];

			$maintenanceLogger->logFromArray( $log );
```

Which allows to add other information to the $log array without having the `$maintenanceHelper` to pull-in script specific information.

Reply:
Ah, I saw this but figured that it was specific to that respective script. Thanks for the note.